feature-utils/poly-analysis/src/analysis.js

```diff
@@ -0,0 +1,150 @@
+import { Telemetry } from "../utils/performance-telemetry.js";
+import { createErrorStatus, createSuccessStatus } from "../utils/status.js";
+import { ZipFile } from "@polypoly-eu/poly-import";
+import InactiveCardsSummary from "./analysis/report/inactive-cards-summary.js";
+
+import MinistoriesStatusAnalysis from "./analysis/report/ministories-status-analysis.js";
+
+export class UnrecognizedData {
+    constructor(analysesResults) {
+        this._activeReportAnalyses = analysesResults
+            .filter(
+                ({ analysis, status }) =>
+                    status.isSuccess &&
+                    analysis.isForDataReport &&
+                    analysis.active
+            )
+            .map(({ analysis }) => analysis);
+
+        const inactiveCardsSummary = new InactiveCardsSummary(analysesResults);
```

Comment:
We can move it later, but all these references to mini stories and cards are making me nervous - this should be _outside_ the scope of _poly-analysis_, all the UI stuff. But we can understand this as an intermediate state I suppose.